ETC.tsv

```diff
@@ -13843,11 +13843,11 @@ ETC_20150729_013844	You can view the login status and location of your friends.
 ETC_20150729_013845	{nl} {nl}[Like] Use {img mouseclick_right 40 40} to
 ETC_20150729_013846	 other characters!
 ETC_20150729_013847	 can be viewed from the Other Character Info window.
-ETC_20150729_013848	Add Magic Amulet on weapons and shields for additional effects.
```

Comment:
Should this be changed back to Magic Amulet? It seems from the other lines that gems and magic amulets are different things.